pages/static/files/component-versions.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
is this just going to be manually updated? What calls componentReferenceImporter to update it?

Reply:
`componentReferenceImporter` is called with with every build and by running `npx gulp bootstrap` or `npx gulp importAll` for devs running amp.dev locally.